datafusion/functions/src/macros.rs

```diff
@@ -160,3 +160,43 @@ macro_rules! downcast_arg {
         })?
     }};
 }
+
+macro_rules! make_abs_function {
```

Comment:
do we really need to introduce this new macros? would be that possible to reuse existing macros like it was before?

Reply:
Yeah, these are not new macros, but from datafusion-physical_expr like https://github.com/apache/arrow-datafusion/pull/9216/files#r1487960276 said.
I think it's also OK to import and reuse the existing macros, but I think in the future if all functions are moved, macros should also move.🤔

Reply:
These are macros for `abs`, I think they should move to math.rs. Also, you need to delete those macros in old file `math_expressions.rs`